CMSgov/managed-care-review
# [MR-2228] Add CMS Dashboard

## Summary
Still in draft but ready for review of api/db work. This is the work that is left:
- [x] wrap in feature flag
- [x] add Cypress tests - added a basic test for one submission. We would need a new approach to dealing with submissions in Cypress via api interaction or seeding the DB to have a robust way to test different combinations of submissions and how they should effect the dashboard.
- [ ] share more UI code between StateDashboard and CMSDashboard components

#### Related issues
https://qmacbis.atlassian.net/browse/MR-2228

#### Test cases covered

see `CMSDashboard.test.tsx` (web tests)
- displays submissions table
- displays submissions table sorted by updatedAt field
- displays submissions table excluding any in progress draft
-  displays each health plan package status tag as expected
- displays program status tags as expected

see `indexHealthPlanPackages.test.ts` for isCMSUser block (api tests)
- returns an empty list if only draft packages exist
- synthesizes the right statuses as a submission is submitted/unlocked/etc
- return a list of submitted packages from multiple states

## QA guidance
Add different types of submission in different statues and check the CMS dashbaord
